crates/lumina_client/src/screens/local_lobby.rs

```diff
@@ -31,9 +40,10 @@ fn spawn_lobby(mut commands: Commands, mut transparency_evw: EventWriter<MainWin
 
             // Spaceship
             builder.spawn((
-                SpaceshipType::Defender.config_info(),
+                SpaceshipType::Assassin.config_info(),
```

Comment:
This should look for the selected spaceship first before defaulting to `SpaceshipType::default()` (yes, I am adding a `Default` derive for `SpaceshipType`)